.github/workflows/js.yml

```
@@ -0,0 +1,35 @@
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: mise
+        uses: jdx/mise-action@v2
+      - name: Change directory to frontend
+        run: cd frontend
+      - name: npm install
+        run: npm install --no-audit
+        working-directory: ./frontend
+      - name: npm lint
+        run: npm run lint --if-present
+        working-directory: ./frontend
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)